P/PAPI/build_tarballs.jl

```diff
@@ -109,14 +109,13 @@ for cuda_version in cuda_versions_to_build, platform in platforms
                                   cuda=tag)
     should_build_platform(triplet(augmented_platform)) || continue
 
-    dependencies = []
+    dependencies = Dependency[
```

Comment:
Did you mean AbstractDependency? Otherwise I think you're converting the runtime dep to normal dep

Reply:
And in fact CUDA_Runtime_jll is installed in the build environment: https://buildkite.com/julialang/yggdrasil/builds/1024#018665b2-a065-493b-9acd-c844ee393ebc/637-651 and below there's a warning about no artifact being available.